contracts/staking/tests/PoSValidatorManagerTests.t.sol

```diff
@@ -372,6 +387,23 @@ abstract contract PoSValidatorManagerTest is ValidatorManagerTest {
         posValidatorManager.resendEndDelegation(delegationID);
     }
 
+    function testResendEndValidation() public override {
```

Comment:
This `override` was brought to this level (from `ValidatorManagerTests`) just for the inclusion of `includeUptime: true`.